dotcom-rendering/src/components/Caption.stories.tsx

```diff
@@ -215,6 +251,13 @@ export const OverlaidWithStars = () => (
 	</Section>
 );
 OverlaidWithStars.storyName = 'when overlaid on stars';
+OverlaidWithStars.decorators = [
+	splitTheme({
+		display: ArticleDisplay.Showcase,
+		design: ArticleDesign.Review,
+		theme: Pillar.News,
+	}),
```

Comment:
It would be great to remove format from the component but we're using it in a few places to determine other styles (padding, width etc). Again, perhaps an argument to store more than just the palette in variables...

For now, I'll extract the formats out so they're more tightly coupled 👍

Reply:
https://github.com/guardian/dotcom-rendering/pull/9413#pullrequestreview-1717254760 <- It would also benefit from having this story change merged in ❤️